static/js/careers-progression.js

```diff
@@ -0,0 +1,298 @@
+      activeIndex = itemNum - 1;
+    } else {
+      activeIndex = activeIndex - 1;
+    }
+
+    goToSlideButton.forEach((element) =>
+      element.setAttribute("aria-current", false)
+    );
+    goToSlideButton[activeIndex].setAttribute("aria-current", true);
+    moveLeft();
+  };
+
+  goToSlideButton.forEach((elem) => {
+    elem.addEventListener("click", (e) => {
+      const index = +elem.dataset.target;
+      let count = Math.abs(index - activeIndex);
+      if (index < activeIndex) {
+        // shift right
+        for (let si = 0; si < count; si++) {
+          moveRight();
+          activeIndex = index;
+        }
+        goToSlideButton[index].setAttribute("aria-current", true);
+        goToSlideButton.forEach((element) =>
+          element.setAttribute("aria-current", false)
+        );
+      } else {
+        // shift left
+        for (let li = 0; li < count; li++) {
+          moveLeft();
+          activeIndex = index;
+        }
+        goToSlideButton[index].setAttribute("aria-current", true);
+        goToSlideButton.forEach((element) =>
+          element.setAttribute("aria-current", false)
+        );
+      }
+    });
+  });
+
+  function moveRight() {
+    shiftToBack(rItem);
+    shiftToRight(activeItem);
+    shiftToActive(lItem);
+
+    rItem = activeItem;
+    activeItem = lItem;
+
+    lItem = carouselItems[lHead--];
+    rHead--;
+
+    if (lHead < 0) {
+      lHead = itemNum - 1;
+    }
+
+    if (rHead < 0) {
+      rHead = itemNum - 1;
+    }
+
+    shiftToLeft(lItem);
+  }
+
+  function moveLeft() {
+    shiftToBack(lItem);
+    shiftToLeft(activeItem);
+    shiftToActive(rItem);
+
+    lItem = activeItem;
+    activeItem = rItem;
+
+    rItem = carouselItems[rHead++];
+    lHead++;
+    if (lHead >= itemNum) lHead = 0;
+    if (rHead >= itemNum) rHead = 0;
+    shiftToRight(rItem);
+  }
+
+  function shiftToRight(elem) {
+    elem.ariaCurrent = "false";
+    elem.style.filter = "blur(4px)";
+    elem.style.transform = `translateX(${radius - itemWidth / 2}px) translateZ(400px)`;
+    elem.style["z-index"] = `${itemNum - 10}`;
+    elem.style.transition = transition;
+  }
+
+  function shiftToLeft(elem) {
+    elem.ariaCurrent = "false";
+    elem.style.filter = "blur(4px)";
+    elem.style.transform = `translateX(${-radius + itemWidth / 2}px) translateZ(400px)`;
+    elem.style["z-index"] = `${itemNum - 10}`;
+    elem.style.transition = transition;
+  }
+
+  function shiftToActive(elem) {
+    elem.ariaCurrent = "true";
+    elem.style.filter = "none";
+    elem.style.transform = "translateX(0) scale(1.25) translateZ(800px)";
+    elem.style["z-index"] = `${itemNum}`;
+    elem.style.transition = transition;
+  }
+
+  function shiftToBack(elem) {
+    elem.ariaCurrent = "false";
+    elem.style.filter = "blur(4px)";
+    elem.style.transform = "translateX(0) translateZ(0)";
+    elem.style["z-index"] = `${itemNum - 20}`;
+    elem.style.transition = transition;
+  }
+}
+
+function setupEdgeHugger() {
```

Comment:
I believe this would be more performant as CSS. Example: https://codepen.io/anthonydillon/pen/JjaVYJR?editors=1100